src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -620,6 +620,11 @@ public virtual NamedTypeSymbol TupleUnderlyingType
         public virtual ImmutableArray<string> TupleElementNames => default(ImmutableArray<string>);
```

Comment:
Either way, I'll leave this for a follow-up PR. I'm adding a note to work items list to remind me.